drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4182,6 +4313,13 @@ void RasterizerSceneGLES3::render_scene(const Transform &p_cam_transform, const
 	// Do depth prepass if it's explicitly enabled
 	bool use_depth_prepass = storage->config.use_depth_prepass;
 
+	// we don't want to do culling in AR
+	bool use_oq = storage->config.use_occlusion_queries;
+
+	use_oq = use_oq && VSG::viewport->viewport_get_update_mode(VSG::viewport->current_viewport_id) != VisualServer::ViewportUpdateMode::VIEWPORT_UPDATE_ONCE;
+	use_oq = use_oq && VSG::viewport->viewport_get_update_mode(VSG::viewport->current_viewport_id) != VisualServer::ViewportUpdateMode::VIEWPORT_UPDATE_DISABLED;
+	use_oq = use_oq && VSG::viewport->viewport_get_allow_occlusion_queries(VSG::viewport->current_viewport_id);
```

Comment:
Pulling the data from the VSG feels a bit icky to me. I think its not actually a problem as these methods aren't exposed through the CommandQueue interface so this shouldn't stall the rendering thread, but it seems like an messy dependency.

I have a feeling that ``use_oq`` should be passed down from the VisualServer into render_scene. That way all this logic can stay with the viewport